cmd/query/app/apiv3/http_gateway.go

```diff
@@ -166,11 +160,16 @@ func (h *HTTPGateway) getTrace(w http.ResponseWriter, r *http.Request) {
 		}
 		request.RawTraces = rawTraces
 	}
-	trc, err := h.QueryService.GetTrace(r.Context(), request)
+	getTracesIter := h.QueryService.GetTraces(r.Context(), request)
+	trc, err := v1adapter.V1TracesFromSeq2(getTracesIter)
```

Comment:
it doesn't make sense to convert to v1 model only to convert it back to OTLP in L113. The benefit of v2 storage here is that we can avoid any unnecessary transforms

Reply:
@yurishkuro yep - will fix. I was just trying to wire up the new query service first.